src/modules/test/MimParser.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#ifndef MIMPARSER_H
+#define MIMPARSER_H
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+struct MimObject
+{
+    std::string name;
+    std::string type;
+    bool desired;
+};
```

Comment:
In the OSConfig C++ code we prefix with m_ class (and struct) members. Please add the same here for consistency